<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1636942700957' stop='1636942749706'>
    <name>Backbase Assessment</name>
    <title>Backbase Assessment</title>
    <test-cases>
        <test-case start='1636942700957' status='failed' stop='1636942749705'>
            <name>Update New Article</name>
            <title>Update New Article</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
            <failure>
                <message>Expected 'Testing Automation Updated' to equal 'Testing Automation'.</message>
                <stack-trace>Error: Failed expectation
                    at /Users/ddsouza/Dropbox/Mac/Desktop/BackbaseAssessment/e2e/pages/updateNewArticle.js:56:34
                    at /usr/local/lib/node_modules/protractor/built/element.js:804:32
                    at ManagedPromise.invokeCallback_ (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:1376:14)
                    at TaskQueue.execute_ (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:3084:14)
                    at TaskQueue.executeNext_ (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:3067:27)
                    at /usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:2927:27
                    at /usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:668:7
                    at processTicksAndRejections (node:internal/process/task_queues:96:5)</stack-trace>
            </failure>
        </test-case>
    </test-cases>
</ns2:test-suite>